scripts/modified-files.sh

```diff
@@ -0,0 +1,23 @@
+if [ -z "$BASH" ]; then
+    echo Bash not installed.
+    exit 1
+fi
+git status >/dev/null 2>&1 || { echo >&2 "Not in a git directory or no git"; exit 1; }
+
+mkdir -p /tmp/swissknife
+touch /tmp/swissknife/modified_files.txt
```

Comment:
Is there any reason I would need to remove this file. Would it exist across runs
@roopakv

Reply:
no need to remove it

Reply:
Okay good to go then